utils/_context/_scenarios/default.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, name: str):
                 "DD_DBM_PROPAGATION_MODE": "service",
                 "SOME_SECRET_ENV": "leaked-env-var",  # used for test that env var are not leaked
                 "DD_EXTERNAL_ENV": "it-false,cn-weblog,pu-75a2b6d5-3949-4afb-ad0d-92ff0674e759",
+                "DD_TRACE_STATS_COMPUTATION_ENABLED": "false",
```

Comment:
From a QA point of view, as by default our customers won';t set this env var, I suggest to add in another scenario some tests, with this env var not set, that assert only traces meant to be dropped are actually dropped, and no more.

Reply:
FYI we have one scenario named TRACING_CONFIG_EMPTY which may be suitable for this